airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/util/Jsons.kt

```diff
@@ -26,7 +27,8 @@ object Jsons : ObjectMapper() {
         setSerializationInclusion(JsonInclude.Include.NON_NULL)
         configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
         configure(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS, true)
-        configure(JsonGenerator.Feature.WRITE_BIGDECIMAL_AS_PLAIN, true)
+        configure(WRITE_BIGDECIMAL_AS_PLAIN, true)
+        configure(JsonNodeFeature.STRIP_TRAILING_BIGDECIMAL_ZEROES, false)
```

Comment:
Note: `STRIP_TRAILING_BIGDECIMAL_ZEROES` directs the Jackson library to not attempt to create scientific notation from numbers with trailing zeros. 
The java big decimal has a cached internal cached text representation, so without this flag turned `false`, a BigDecimal created inside a json `DecimalNode` of 100 would be created as (abbv.)
```
{
  val: 100
  textCache: "1E+2"
}
```

With this change it is created as 
```
{
  val: 100
  textCache: "100"
}
```

So a simple `.toString()` would return "100".